### PR TITLE
README - don't update to PIP version 10

### PR DESCRIPTION
**Pull request by JanneKiiskila**
We should really mention that don't update to `pip` version 10.0.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## Important note - do not use Pip version 10.0!
+
+Pip version 10 is incompatible with current versions of `mbed cli`.
```
Comment (cmonr) [inline README.md]: I would suggest adding _why_ upgrading to pip v10 is not supported. 

Mainly, this PR: https://github.com/pypa/pip/pull/4700

**Comment by cmonr** [comment]
@theotherjimmy

**Comment by screamerbg** [comment]
@JanneKiiskila Thanks for the PR. Could you please try to update mbed CLI to v1.5.1 and let me know if this is now fixed?

**Comment by screamerbg** [comment]
@JanneKiiskila Can we close this if resolved?

**Comment by screamerbg** [comment]
Just merged the other request. Will close this one as it's now obsolete (i.e. pip10 is supported)

**Comment by JanneKiiskila** [comment]
Nope, you should still tell people we have this limitation. If you want pip version 10, you must upgrade to mbed cli version 1.5.1 o newer. That issue still remains.

**Comment by screamerbg** [comment]
https://github.com/ARMmbed/mbed-cli#mbed-cli-and-pip-compatibility. Its already there

**Pull request closed without merging**